docs/example-react-router.mdx

```diff
@@ -133,8 +133,45 @@ test('rendering a component that uses useLocation', () => {
 
 ## Testing Library and React Router v5
 
-In React Router v5, you need to pass the history object as a whole to the Route
-component.
+Given your App uses the correct React Router v5 API.
```

Comment:
I think we can remove this line as it doesn't really add anything to the reader :)

Reply:
I removed the line